ompi/mca/coll/base/coll_base_comm_select.c

```diff
@@ -565,8 +565,10 @@ static int query(const mca_base_component_t * component,
 {
     *module = NULL;
     if (3 == component->mca_type_major_version &&
-        0 == component->mca_type_minor_version &&
-        0 == component->mca_type_release_version) {
+        ((0 == component->mca_type_minor_version &&
+          0 == component->mca_type_release_version) ||
+         (1 == component->mca_type_minor_version &&
+          0 == component->mca_type_release_version))) {
 
         return query_3_0_0((const mca_coll_base_component_3_0_0_t *)component, comm, priority, module);
```

Comment:
Since you have bumped to 3.1.0, please rename this function to `query_3_1_0` -- the convention is to name everything according to the highest required version.

Reply:
of course